code/game/turfs/walls/walls.dm

```diff
@@ -43,6 +43,8 @@
 	var/list/noblend_turfs = list(/turf/closed/wall/mineral, /turf/closed/wall/almayer/research/containment) //Turfs to avoid blending with
 	var/list/blend_objects = list(/obj/structure/machinery/door, /obj/structure/window_frame, /obj/structure/window/framed) // Objects which to blend with
 	var/list/noblend_objects = list(/obj/structure/machinery/door/window) //Objects to avoid blending with (such as children of listed blend objects.
+	///Is someone currently leaning on it, and who
+	var/mob/living/leaned_on_by
```

Comment:
this looks like it could be used as such but you only use it as human later down